addons/point_of_sale/models/pos_order.py

```diff
@@ -187,7 +187,7 @@ def _process_payment_lines(self, pos_order, order, pos_session, draft):
                 'name': _('return'),
                 'pos_order_id': order.id,
                 'amount': -pos_order['amount_return'],
-                'payment_date': fields.Datetime.now(),
+                'payment_date': pos_order['creation_date'].replace('T', ' ')[:19],
```

Comment:
```suggestion
                'payment_date': order.create_date,
```

Reply:
or date_order

Reply:
`create_date` won't be right as the payment could be from yesterday and the order was just synched from the pos client.

`date_order` is correct :+1: